sycl/test/regression/sycl-include-gnu17.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -std=gnu++14 -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %clangxx -std=gnu++17 -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
@premanandrao, do we still need test which checks compilability with `gnu++17`?

Reply:
It was added very early I believe we were not compiling something from the standard header files.  I think keeping it is useful as a sanity test.

Reply:
Ok, thanks Prem!